src/index.js

```diff
@@ -0,0 +1,118 @@
+import ui from './modules/ui';
+import Gameboard from './modules/data/gameboard';
+import Player from './modules/data/players';
+
+   const playerStorage = new Array(100).fill(null);
+   const computerStorage = new Array(100).fill(null);
+
+   const playerBoard = Gameboard();
+   const computerBoard = Gameboard();
+   
+   const player = Player('player', playerBoard, playerStorage);
+   const computer = Player('computer', computerBoard, computerStorage);
+
+   let turn = 'player';
```

Comment:
This variable is never used. You should try to incorporate it as means to authenticate if it is the player turn  or the computer turn. It comes to me that you could use a while statement to manage your turns and make the appropiate moves depending on the turn string value. But something that prevents that bug from happening, where I can click an used tile and it will trigger the comp turn.